### PR TITLE
Create gaza-specific scavenger-hunt

### LINKED ISSUE #4: Add Gaza specific scavenger hunt
The [master-reference scavenger hunt](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/general/github-scavenger-hunt.md) links to a non-existent page in this repo.

Should add a gaza specific version of the [london-programme](https://github.com/foundersandcoders/london-programme/blob/master/github-scavenger-hunt.md); many of the things in London will not be relevant for Gaza.

### PR DESCRIPTION
**Pull request by eliascodes**
Fixes #4 

Many items removed relative to the London version since they are not relevant here (yet).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,3 +42,17 @@ Please read the [CONTRIBUTING](./CONTRIBUTING.md) guidelines.
 [Master-Reference](https://github.com/foundersandcoders/master-reference): Contains the full curriculum, as well as records and discussions about all aspects of the curriculum
```
Comment (rebeccaradding) [inline README.md]: Should we remove these contributing guidelines and add a link to the `consensus-decisions` document to the README?

Comment (eliascodes) [inline README.md]: Sure, but let's do that in a different PR. Have opened  #6
